Automattic/jetpack
# My Jetpack: put My Jetpack menu item at top of Jetpack menu section

This PR puts My Jetpack menu item at top of Jetpack menu section. Also, when the site is disconnected, My Jetpack shows the connection screen allowing the user get reconnected to the site. No hard redirection.

Fixes #

#### Changes proposed in this Pull Request:

* My Jetpack: put My Jetpack menu item at top of Jetpack menu section

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Install and activate only one stand-alone plugin (e.g. Boost or Backup)
* Make sure the Jetpack menu is added and My Jetpack is the first item
* ~~Confirm that when you visit My Jetpack you are redirected to the interstitial page of the plugin you have active (e.g. `#/add-boost`)~~ Confirm that when you visit the My Jetpack page you see the regular dashboard with the product cards
* Activate a second stand-alone plugin
* Confirm that when you visit the My Jetpack page you see the regular dashboard with the product cards
* Confirm that adding products works from there
* Test differentt combinations and actions you think could be tricky

before | after
------|--------
<img width="333" alt="image" src="https://user-images.githubusercontent.com/77539/155235066-d68792ac-c4dd-47ec-bcff-d1812c839f4e.png"> | <img width="363" alt="image" src="https://user-images.githubusercontent.com/77539/155234981-f2de26f0-6962-498d-b04b-35dd10b39b46.png">
